frontend/public/learn/Interactive-UI-Elements.elm

```diff
@@ -46,38 +51,40 @@ and then dive into how it works.
 We are going to make three synced checkboxes. Changing one of them will change
 the two others:
 
-|]
+"""
 
-rest = [markdown|
+rest : Element
+rest = Markdown.toElement """
 
 Here is the code we need to make that happen. Don't worry about the details too
 much yet. This is more to get a feel for the API so we know what we are working
 towards when we dive into the details:
 
 ```haskell
-import Graphics.Input (Input, input, checkbox)
+import Graphics.Input (checkbox)
+import Signal
 
-check : Input Bool
-check = input False
+check : Signal.Channel Bool
+check = Signal.channel False
 
 displayBoxes : Bool -> Element
 displayBoxes isChecked =
-    let box = checkbox check.handle identity isChecked
+    let box = checkbox (Signal.send check) isChecked
     in  flow right [ box, box, box ]
 
 main : Signal Element
-main = displayBoxes <~ check.signal
+main = displayBoxes <~ Signal.subscribe check
```

Comment:
The `<~` operator is not imported on line 65; suggest `Signal.map displayBoxes (Signal.subscribe check)`.